docs/user-guide/combine.md

```diff
@@ -1,364 +0,0 @@
-# Combining rules
```

Comment:
[note] The 1st half of this file has been summarised in the new customisation guide. The 2nd half is no longer relevant as it is about the deprecated rules.